eclair-core/src/test/scala/fr/acinq/eclair/channel/FuzzySpec.scala

```diff
@@ -81,107 +81,101 @@ class FuzzySpec extends TestkitBaseClass with StateTestsHelperMethods with Loggi
       bob ! WatchEventConfirmed(BITCOIN_FUNDING_DEPTHOK, 400000, 42, fundingTx)
       alice2blockchain.expectMsgType[WatchLost]
       bob2blockchain.expectMsgType[WatchLost]
-      awaitCond(alice.stateName == NORMAL)
-      awaitCond(bob.stateName == NORMAL)
+      awaitCond(alice.stateName == NORMAL, 1 minute)
```

Comment:
1 minute looks huge!

Reply:
Yes, was kind of a desperate measure. I should be able to reduce/remove once I figure out the root cause.